manifests/t/Task/Task/v3.29.1/Task.Task.installer.yaml

```
@@ -0,0 +1,31 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+PackageIdentifier: Task.Task
+PackageVersion: v3.29.1
+InstallerLocale: en-US
+InstallerType: zip
+ReleaseDate: "2023-08-26"
+Installers:
+  - Architecture: x86
+    NestedInstallerType: portable
+    NestedInstallerFiles:
+      - RelativeFilePath: task.exe
+    InstallerUrl: https://github.com/go-task/task/releases/download/v3.29.1/task_windows_386.zip
+    InstallerSha256: 42130ca4d48d93be99b6942ee42716f1401333b7a4ecaa54caafc5f32ec5257c
+    UpgradeBehavior: uninstallPrevious
+  - Architecture: arm64
+    NestedInstallerType: portable
+    NestedInstallerFiles:
+      - RelativeFilePath: task.exe
+    InstallerUrl: https://github.com/go-task/task/releases/download/v3.29.1/task_windows_arm64.zip
+    InstallerSha256: 185c8dcac498d42f39dbac9aad37966b1d27c647f2dfb843f2b97a4685a892fc
+    UpgradeBehavior: uninstallPrevious
+  - Architecture: x64
+    NestedInstallerType: portable
+    NestedInstallerFiles:
+      - RelativeFilePath: task.exe
+    InstallerUrl: https://github.com/go-task/task/releases/download/v3.29.1/task_windows_amd64.zip
+    InstallerSha256: 9ad3a6fa920b85466545bcbc6a192118f39465757432c835bd0b58f75d862991
+    UpgradeBehavior: uninstallPrevious
```

Hi @andreynering, these are the actual hashes.

```suggestion
  - Architecture: x86
    NestedInstallerType: portable
    NestedInstallerFiles:
      - RelativeFilePath: task.exe
    InstallerUrl: https://github.com/go-task/task/releases/download/v3.29.1/task_windows_386.zip
    InstallerSha256: 09FB1A0469C5A75A64AA26623171E92A84188DCB3CF4AADA3D30390979C64124
    UpgradeBehavior: uninstallPrevious
  - Architecture: arm64
    NestedInstallerType: portable
    NestedInstallerFiles:
      - RelativeFilePath: task.exe
    InstallerUrl: https://github.com/go-task/task/releases/download/v3.29.1/task_windows_arm64.zip
    InstallerSha256: 6CE71F4DE3964EA8D86400523B1C2FEF9E8EF8DB5D0812FD54045A5F2E71B050
    UpgradeBehavior: uninstallPrevious
  - Architecture: x64
    NestedInstallerType: portable
    NestedInstallerFiles:
      - RelativeFilePath: task.exe
    InstallerUrl: https://github.com/go-task/task/releases/download/v3.29.1/task_windows_amd64.zip
    InstallerSha256: AAEA73A76F50BF9AB49301256E42314101980F249E750082D25F6ED9D16C30C3
    UpgradeBehavior: uninstallPrevious
```

---

Thanks!

I generated this locally with a different Go version, which explains the confusion.